lib/plugins/inventory.js

```diff
@@ -391,15 +391,20 @@ function inject (bot, { hideErrors }) {
   }
 
   function tradeMatch (limitItem, targetItem) {
-    return targetItem.type === limitItem.type && targetItem.count >= limitItem.count
+    return (
+      targetItem !== null &&
+      limitItem !== null &&
+      targetItem.type === limitItem.type &&
+      targetItem.count >= limitItem.count
+    )
```

Comment:
Sometimes, `tradeMatch` is called with a null `targetItem`. If there is nothing in the `targetItem` slot, then the trade requirements cannot be satisfied, so return false in that case.